src/Raven.Server/Config/Categories/MemoryConfiguration.cs

```diff
@@ -39,5 +39,10 @@ public MemoryConfiguration()
         [SizeUnit(SizeUnit.Megabytes)]
         [ConfigurationEntry("Memory.MaxFreeCommittedMemoryToKeepInMb", ConfigurationEntryScope.ServerWideOnly)]
         public Size MaxFreeCommittedMemoryToKeepInMb { get; set; }
+
+        [Description("EXPERT: Use 'RSS' instead of 'memory.usage_in_bytes minus Shared Clean Memory' value to determine machine memory usage in docker instance. Applicable only with environment variable RAVEN_IN_DOCKER is set to 'true' and only when running on Linux. Will use configuration option 'Memory.LowMemoryLimitInMb' with RavenDB process RSS value. Default: true.")]
+        [DefaultValue(true)]
+        [ConfigurationEntry("Memory.UseRssInsteadOfMemUsageInContainer", ConfigurationEntryScope.ServerWideOnly)]
+        public bool UseRssInsteadOfMemUsageInContainer { get; set; }
```

Comment:
I don't like this option. Too tailored.
Instead, let's just make it possible for the admin to make the selection between the option. Regardless of docker or not

Reply:
https://github.com/ravendb/ravendb/pull/9700